docs/painless/painless-contexts/painless-score-context.asciidoc

```diff
@@ -24,4 +26,32 @@ score to documents returned from a query.
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+The following query will find all unsold seats, with lower 'row' values
+scored higher.
+
+[source,js]
+--------------------------------------------------
```

Comment:
It looks like this one is failing when we run the docs tests. I expect the index is missing.

Reply:
This should be fixed now